store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorage.java

```diff
@@ -598,23 +609,68 @@ private boolean changeGraphId(final String graphId, final String newGraphId, fin
                 }
             }
 
-            final GraphConfig configWithNewGraphId = new GraphConfig.Builder()
-                    .json(new GraphSerialisable.Builder().graph(graphToMove).build().getConfig())
-                    .graphId(newGraphId)
-                    .build();
+            //Update Tables
+            String storeClass = graphToMove.getStoreProperties().getStoreClass();
+            if (nonNull(storeClass) && storeClass.startsWith(AccumuloStore.class.getPackage().getName())) {
```

Comment:
This approach isn't super portable, will it work for all other stores? e.g. HBase

Reply:
No it won't, this is a code block is for Accumulo with Accumulo specific commands.

Reply:
Maybe there should be a smarter solution then? Otherwise this will just be a big if statement within the FederatedStore to make sure it works for all other store types

Reply:
Agreed. But I don't know if this problem exist with other stores. (probably does)
In the interim I wanted to get a 95% solution in.
I'm happy to add a comment/new ticket